core-bundle/src/Resources/config/services.yml

```diff
@@ -578,10 +572,23 @@ services:
     contao.security.authentication_failure_handler:
         class: Contao\CoreBundle\Security\Authentication\AuthenticationFailureHandler
         arguments:
-            - '@http_kernel'
+            - '@?logger'
```

Comment:
This is marked as optional dependency but it is a mandatory constructor argument in the class.

Reply:
fixed in a507cf7a1e81c189e37294aa3346b3b2f0b7f77f